src/test/java/uk/gov/justice/digital/delius/jpa/standard/entity/EventTest.java

```diff
@@ -1,51 +1,88 @@
 package uk.gov.justice.digital.delius.jpa.standard.entity;
 
-import org.junit.Test;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class EventTest {
-    @Test
-    public void hasCpsPackWhenHasAlfrescoIdAndNotDeleted() {
-        assertThat(Event
-                .builder()
-                .cpsAlfrescoDocumentId("1334")
-                .cpsSoftDeleted(0L)
-                .build().hasCpsPack()).isTrue();
-    }
-    @Test
-    public void hasCpsPackWhenHasAlfrescoIdAndNotDeletedFlagAbsent() {
-        assertThat(Event
-                .builder()
-                .cpsAlfrescoDocumentId("1334")
-                .cpsSoftDeleted(null)
-                .build().hasCpsPack()).isTrue();
-    }
-    @Test
-    public void doesNotHaveCpsPackWhenHasAlfrescoIdAndNotDeletedTrue() {
-        assertThat(Event
-                .builder()
-                .cpsAlfrescoDocumentId("1334")
-                .cpsSoftDeleted(1L)
-                .build().hasCpsPack()).isFalse();
-    }
+    @Nested
+    class CPSPack {
+
+        @Test
+        public void hasCpsPackWhenHasAlfrescoIdAndNotDeleted() {
+            assertThat(Event
+                    .builder()
+                    .cpsAlfrescoDocumentId("1334")
+                    .cpsSoftDeleted(0L)
+                    .build().hasCpsPack()).isTrue();
+        }
+
+        @Test
+        public void hasCpsPackWhenHasAlfrescoIdAndNotDeletedFlagAbsent() {
+            assertThat(Event
+                    .builder()
+                    .cpsAlfrescoDocumentId("1334")
+                    .cpsSoftDeleted(null)
+                    .build().hasCpsPack()).isTrue();
+        }
 
-    @Test
-    public void doesNotHaveCpsPackWhenDoesNotHaveAnAlfrescoId() {
-        assertThat(Event
-                .builder()
-                .cpsAlfrescoDocumentId("")
-                .cpsSoftDeleted(0L)
-                .build().hasCpsPack()).isFalse();
+        @Test
+        public void doesNotHaveCpsPackWhenHasAlfrescoIdAndNotDeletedTrue() {
```

Comment:
should it be soft deleted in the test name?

Reply:
possibly, although I didn't change these test, just moved them inside a nested test class (which is no defence I suppose)